### ISSUE TITLE
[FEATURE] Add a different stacking rule to tooltips.

### ISSUE THREAD

**Issue by DashieTM** [elkowar/eww#555]
### Description of the requested feature

Currently hovering over a widget with tooltip forces said tooltip to use the same stacking rule as the widget. This leads to strange behavior where a tooltip appears behind a window.
![1](https://user-images.githubusercontent.com/72016555/188946018-1ce9dbc9-4b75-46b7-bd08-d46572aa64eb.png)
![2](https://user-images.githubusercontent.com/72016555/188946412-473a3a8c-daa1-4fa8-8834-a9b261a1bfeb.png)
Hard to notice, but there is a tooltip behind the terminal.

My suggestion is, if possible, a flag that allows a tooltip to have a different stacking rule.
Widget -> bg
tooltip -> fg

If this is a bug instead, aka it works for other window managers etc then I am happy to report it instead. Currently using Hyperland.

### Proposed configuration syntax

:tooltip "something"
:tooltip-stacking true/false

**Comment by devgioele** [comment]
Also the orientation is the same for the tooltip. When using a vertical bar, tooltips show vertically on the bar itself.
More customization of the tooltip is necessary

**Comment by blasseye** [comment]
I agree, I have my eww bar at the bottom, and the tooltips are displayed below the screen... Impossible to put them at the top.
It's a great idea for a feature.
